index.js

```diff
@@ -61,6 +61,12 @@ module.exports = {
         { label: 'External URL', value: 'external', showFields: [ 'externalUrl' ] }
       ]
     },
+    {
+      name: 'ignoreQuerystring',
```

Comment:
`ignoreQueryString` please (capitalize the S throughout).

Reply:
Doe.